### ISSUE TITLE
Sewers are quite too common (And so do the other new WOTC maps)

### ISSUE THREAD

**Issue by Anxoczt** [long-war-2/lwotc#405]
After raking tens of hours of LWOTC i started to notice that some map repeat abit too much. Mainly the sewers, which with my luck is like around 50% of maps i get are sewers or lost cities.

And this is really odd, i might be just unlucky but i just get it abit way too much, i didnt mind it in vanilla WOTC but with the increased mission count.... 

And both of these maps are annoying in their own way. In lost cities you cannot use explosives and the abandoned cities are total deathtraps when enemies get to yellow or red alert. And sewers are choke point fights at random distances with no flanking and snipers are heavily penalised here because obstruction almost everywhere.

I wouldnt mind if only specific missions could have them but majority of generic missions can get in to the sewers or lost cities (Though non-supply raid missions in those cities are somewhat rare but i managed to get troop collum ambush and destroy relay in them). More than half of Advent unit ambush happen in sewers for unknown reason so when i get to ambush them in a city i am like kissing the ground since now i can snipe them fromrooftops or soften them up with grenades all i want.

Is it just me and my bad RNG or is anyone else also experiencing this ?.

**Comment by jmartinez989** [comment]
I have noticed this as well. In particular I see the tunnel maps mainly on destroy the alien relay and troop column missions.

I do agree that they shouldn't show up as much as they do but at the same time these maps also somewhat deal with the problem of dfa sharpshooters. I definitely think that the tunnel maps should come down in frequency but am ok with seeing them for specific missions at least a bit more than usual.

**Comment by pledbrook** [comment]
What's your output from the `X2DebugDeck PlotTypes` console command?

**Comment by Anxoczt** [comment]
Here it is

![plots](https://user-images.githubusercontent.com/50877085/62937861-0f5a4e80-bdce-11e9-8089-6d2ef0d637db.png)

**Comment by cshen6** [comment]
Yes it seems a few mission types are more likely to have sewer map, the troop column ambush is one of them; and another one is resistance evac, where there is no time limit but enemy reinforcement every 2 turns & resistance evac first then wait 4 turns to evac soldiers.
Other than that, most of the maps are city maps. 

The main downside of sewer map is lack of high ground & clear sight for sharpshooters, and there is no hack tower for specialist. I feel the better option is to display the map type together with the mission, if that is possible.

Tbh, Vanilla WOTC has a even higher chance of rolling sewer or lost map, which renders specialist & sharpshooters much less useful.

**Comment by pledbrook** [comment]
Well, if the output of that command is to be believed, then Anxoczt is getting an even spread of plot types in missions. Not sure if I actually believe that, though. I've seen the numbers go all over the place.

We will probably have to rip out the vanilla plot selection and implement our own.

**Comment by pledbrook** [comment]
> Tbh, Vanilla WOTC has a even higher chance of rolling sewer or lost map, which renders specialist & sharpshooters much less useful.

As far as I understand it (which is limited), the plot type count carries over from all previous missions, including the ones prior to WOTC. So to get the numbers back evenly spread, you end up seeing the new plot types more frequently. You can mitigate that by deleting the _profile.bin_ file which contains the deck state (I think).

**Comment by cshen6** [comment]
> > Tbh, Vanilla WOTC has a even higher chance of rolling sewer or lost map, which renders specialist & sharpshooters much less useful.
> 
> As far as I understand it (which is limited), the plot type count carries over from all previous missions, including the ones prior to WOTC. So to get the numbers back evenly spread, you end up seeing the new plot types more frequently. You can mitigate that by deleting the _profile.bin_ file which contains the deck state (I think).

Thank you for clarification! Maybe that is why I am seeing almost no sewer maps at first, then gradually increases in my recent play.
Still, my vanilla WOTC playthroughs are infested with sewer or lost maps all the time -- so really glad to back to enjoy the big city maps in LW2 :-)

**Comment by ToastedAmphibian** [comment]
Can I second the "Display map type on mission briefing" suggestion?

**Comment by pledbrook** [comment]
Have you tried [Show Enemies and Terrain on Mission Planning](https://steamcommunity.com/sharedfiles/filedetails/?id=1183583781)?

**Comment by pledbrook** [comment]
See #432 for issue to add plot (map) type to the squad select screen. That's the only place it's really relevant I think.

**Comment by Grobobobo** [comment]
Already made a bandaid fix for this by removing the subway, and we're not really doing anyting deeper.
